assets/src/edit-story/app/media/utils/createResource.js

```diff
@@ -32,7 +32,8 @@ import getTypeFromMime from './getTypeFromMime';
  * Attribution object
  *
  * @typedef {Attribution} Attribution
- * @property {Author} author The author of the media object.
+ * @property {?Author} author The optional author of the media object.
+ * @property {string} registerUsageUrl The url to register the media usage.
```

Comment:
This should be ?string as Coverr doesn't have registerUsage.

Reply:
Didn't we agree that we'll still issue a registerUsageUrl for all providers and then just ignore it on the API side? This allows us to keep the client simpler.

Reply:
I don't think so, I think we agreed we'd use the same approach for registerUsage across all providers, but not that we'd be issuing registerUsage calls from the client for all providers. Whether a provider needs registerUsage can be configured as part of the provider configuration json for example.

Reply:
As discussed offline, made the url nullable.